Configuration/AlCa/python/autoCond.py

```diff
@@ -42,9 +42,9 @@
     # GlobalTag for Run3 data relvals (prompt GT) - identical to 124X_dataRun3_Prompt_v5 but with snapshot at 2022-10-04 14:19:51 (UTC)
     'run3_data_prompt'             : '124X_dataRun3_Prompt_frozen_v5',
     # GlobalTag for Run3 offline data reprocessing - snapshot at 2022-11-01 12:00:00 (UTC)
```

Comment:
Just checking: would you like to update also the snapshot time in the comment line (here and below), as you do in the other versions of this PR? (If you want to do so and you only modify the comment lines, I won't retrigger tests and signature, and I'll just merge the PR as it is)

Reply:
Hi @perrotta , indeed, I forgot to update the comment line. But there is another PR to 124X in the pipeline (to update the online GTs) and I can update the comment lines then, if it is ok with you.

Reply:
yes, it would be the backport of #40367 
It sounds good. Let me update this PR then now.